atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/service/WriteBatchingTransactionService.java

```diff
@@ -123,7 +123,7 @@ public void close() {
      *
      * Retrying does theoretically mean that in the worst case with N transactions in our batch, we may actually
      * require N calls to the database, though this is extremely unlikely especially because of the semantics of
-     * {@link TimelockService#startIdentifiedAtlasDbTransaction()}.
+     * {@link TimelockService#startIdentifiedAtlasDbTransactionBatch(int)}.
```

Comment:
Just want to check that this simple substitution still makes sense in the context above.